Lib/webbrowser.py

```diff
@@ -265,9 +297,11 @@ def open(self, url, new=0, autoraise=True):
                 action = self.remote_action_newwin
             else:
                 action = self.remote_action_newtab
+        elif new == 3:
+            action = self.remote_action_incognito
         else:
             raise Error("Bad 'new' parameter to open(); " +
-                        "expected 0, 1, or 2, got %s" % new)
+                        "expected 0, 1, 2, or 3, got %s" % new)
```

Comment:
Shall we take this opportunity to f-string it? 
```suggestion
                        f"expected 0, 1, 2, or 3, got {new}")
```

Reply:
i gave also access to you for this repo...
probably pep08 in addition ...